datafusion/sql/Cargo.toml

```diff
@@ -57,6 +57,7 @@ ctor = { workspace = true }
 datafusion-functions = { workspace = true, default-features = true }
 datafusion-functions-aggregate = { workspace = true }
 datafusion-functions-window = { workspace = true }
+datafusion-functions-nested = { workspace = true }
```

Comment:
Added as dev dependency only. Required for `make_array` to test the following unnest case:

SELECT unnest(make_array(1, 2, 2, 5, NULL)) as u1